code/game/objects/structures/droppod.dm

```diff
@@ -17,26 +22,45 @@ GLOBAL_LIST_INIT(blocked_droppod_tiles, typecacheof(list(/turf/open/space/transi
 	max_integrity = 50
 	flags_atom = PREVENT_CONTENTS_EXPLOSION
 	coverage = 75
-	var/mob/occupant
+	buckle_flags = CAN_BUCKLE|BUCKLE_PREVENTS_PULL
+	buckle
```

Comment:
?

Reply:
oh tahst where that wierd type came from